diff --git a/node_modules/react-native-screens/android/build.gradle b/node_modules/react-native-screens/android/build.gradle
index 58fe0e4..a6e0e66 100644
--- a/node_modules/react-native-screens/android/build.gradle
+++ b/node_modules/react-native-screens/android/build.gradle
@@ -38,6 +38,10 @@ def reactNativeArchitectures() {
 }
 
 android {
+    namespace "com.swmansion.rnscreens"
+    buildFeatures {
+        buildConfig true
+    }
     compileSdkVersion safeExtGet('compileSdkVersion', 28)
 
     // Used to override the NDK path/version on internal CI or by allowing
