diff --git a/node_modules/react-native-gesture-handler/android/build.gradle b/node_modules/react-native-gesture-handler/android/build.gradle
index fa5ffe4..b6c6edc 100644
--- a/node_modules/react-native-gesture-handler/android/build.gradle
+++ b/node_modules/react-native-gesture-handler/android/build.gradle
@@ -120,8 +120,13 @@ repositories {
 }
 
 android {
+    namespace "com.swmansion.gesturehandler"
     compileSdkVersion safeExtGet("compileSdkVersion", 28)
 
+    buildFeatures {
+        buildConfig true
+    }
+
     // Used to override the NDK path/version on internal CI or by allowing
     // users to customize the NDK path/version from their root project (e.g. for M1 support)
     if (rootProject.hasProperty("ndkPath")) {
